### ISSUE TITLE
Add item images

### ISSUE THREAD

**Issue by maplethorpej** [maplethorpej/packstack#64]
User request:

Ability to add pictures, or at least link to pictures hosted elsewhere with an option to have display as thumbnails on each item row.  If not that, then ability to add picture of pack (or trail, or anything) within a pack lists overview.  I understand if there's space/funding/hosting concerns here, but would be like a cherry on top of everything else you have going here with it.

**Comment by Callisto13** [comment]
My opinion: having images on rows would be a bit of a mess. The only way to have it not be a mess, would be to have them in like cute small circle thumbnails, at which point the size makes them useless. We already have a field for product link, which will have a pic of the item.

I vote no for this feature, on the site version at least.

**Comment by via-justa** [comment]
I can think of 2 solutions for that:
- Resize the image on upload so it can be in the expected max size
- Create a collapsable item view that can hold the picture. That option would also allow to adding more item fields without changing the main interface
I personally would love to have the pictures as I got many items of the same category that I remember what they are just by looking at them.

**Comment by nordmanbp** [comment]
I agree with Callisto13 and vote no on this, or at least very low priority.
